src/EasyList/Validate.cs

```diff
@@ -0,0 +1,135 @@
+﻿using EasyList;
+using EasyList.DataModels;
+using Sharprompt;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace EasyList
+{
+    public class Validate
+    { 
+        private static bool IsErrorFree(IEnumerable<string> errors)
```

Comment:
Method name implies it checks if the todo is error free but thats not what it does

Reply:
What do you suggest ?

Reply:
First of all this method should be "PrintErrors" because the if errors.any can be checked outside the code. This method does two things, it checks if there are any errors and then prints all the errors at the same time it returns a boolean. If you do something like `if(todoValidator.IsValid){todoValidator.PrintErrors}` it would make a bit more sense and also follow principles such as SRP in this case the method should only have one responsibility, one job.